### ISSUE TITLE
Integrate bluepill 

### ISSUE THREAD

**Issue by gbasile** [fastlane/fastlane#8162]
### New Issue Checklist

- [✔️ ] Updated fastlane to the latest version
- [✔️ ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Bluepill is a tool introduced by Linkedin that  run iOS tests in parallel using multiple simulators.
I would like to have this functionality in fastlane. 
I'm open to contribute to the implementation of the action, but according to the guidelines I should check first if you it will be included.

**Comment by hjanuschka** [comment]
i think we have a good tool for a similiar use case _snapshot_, however i can understand your feature-request, and bluepill looks amazing, but right now i think its  a good example for a _fastlane-plugin_.

see the guide here: https://docs.fastlane.tools/plugins/create-plugin/
if there is a large user-base for the plugin, it may become part of the fastlane-core.


let me know if you need any further help.

**Comment by KrauseFx** [comment]
Actually I'd very much love this. The problem is that it's not based on Ruby, however it would make sense to take a closer look on how they implemented certain things to make use of multiple simulators at the same time for _snapshot_. _snapshot_ is pretty slow right now for that reason, however we'd have to completely rewrite _snapshot_ to make use of it.

Definitely something to keep in mind, however not something we can work on right now. Feel free to work on a _fastlane_ plugin or PR for this 👍

**Comment by ohayon** [comment]
Going to close this issue for now. But feel free to continue conversation here. Bluepill looks really awesome and it would be amazing to have this in a plugin! 🐙 💯
